client/app/scripts/actions/app-actions.js

```diff
@@ -635,13 +635,15 @@ function updateFromNodesDeltaBuffer(dispatch, state) {
 }
 
 export function clickResumeUpdate() {
-  return (dispatch, getServiceState) => {
+  return (dispatch, getState) => {
     dispatch({
       type: ActionTypes.CLICK_RESUME_UPDATE
     });
+    // TODO: Find a better way to do this (see the comment above).
+    const state = getState().scope || getState();
```

Comment:
This should not be necessary if the action is being called from inside the `<Namespace />` by a component, and the entirety of Scope should be in there. I will pull this down and take a look.